.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: python
 
 python:
-    - "2.7"
-    - "3.4"
-    - "3.5"
-    - "3.6"
+  - "2.7"
+  - "3.7"
+  - "3.6"
+
+dist: xenial
 
 install:
-    - pip install -r requirements.pip
-    - pip install -r requirements-dev.pip
-    - pip install -e .
+  - pip install -r requirements.pip
+  - pip install -r requirements-dev.pip
+  - pip install -e .
+
+script: pytest --cov-fail-under=99 -n auto
```

Comment:
when running scripts standalone you should just write "pytest" <enter> and it will take all the settings from setup.cfg, except cov-fail-under and xdist usage

but I will do as you wish, changing now